ui/src/App.tsx

```diff
@@ -13,9 +13,11 @@ const useStyles = makeStyles(() => createStyles({
 
 export const App = (): ReactElement => {
   const [selected, setSelected] = useState<number>(0);
-  const { data: mountPoint } = useMountPoint();
+  const { user, os } = useMountPoint();
   const classes = useStyles();
-  const shouldDialogOpen = !mountPoint || mountPoint === '';
+  const  ddClient  = useDDClient();
```

Comment:
```suggestion
  const ddClient = useDDClient();
```

Curious to find out why the linter would not report this (minor) formatting issue? If I run `(cd ui; npm run lint)`, the result seems to be clean. Could this be related to the custom `rules` in `.eslintrc.js`?